<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtCMBCreateSimpleGeometry</class>
 <widget class="QDialog" name="qtCMBCreateSimpleGeometry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Basic Geometry</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="geometryType">
     <item>
      <property name="text">
       <string>Rectangle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ellipse</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="rectanglePage">
      <widget class="QDoubleSpinBox" name="xMinimum">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="yMinimum">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Minimum coordinates</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>171</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Maximum coordinates</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="xMaximum">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>70</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="yMaximum">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QSpinBox" name="baseResolution">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>160</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="heightResolution">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>160</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>159</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Base resolution</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>159</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Height resolution</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="ellipsePage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Center (Xc, Yc)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Resolution</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(x-x&lt;span style=&quot; vertical-align:sub;&quot;&gt;c&lt;/span&gt;)&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;/A&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;+(y-y&lt;span style=&quot; vertical-align:sub;&quot;&gt;c&lt;/span&gt;)&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;/B&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;=1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="yRadius">
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="xRadius">
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="ellipseResolution">
         <property name="minimum">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>X radius (A)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="yCenter"/>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="xCenter"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Y radius (B)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qtCMBCreateSimpleGeometry</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qtCMBCreateSimpleGeometry</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
